test/unit/processing/siti-agri/build-payment-requests.test.js

```diff
@@ -1,37 +1,37 @@
-const buildPaymentRequests = require('../../../app/parse-siti-payment-file/build-payment-requests')
+const { GBP } = require('../../../../app/currency')
+const buildPaymentRequests = require('../../../../app/processing/siti-agri/build-payment-requests')
+const { Q4 } = require('../../../../app/schedules')
+const { sfiPilot } = require('../../../../app/schemes')
 
 global.console.error = jest.fn()
```

Comment:
Going to create a ticket in backlog to strip out logging from tests

Reply:
Yes good idea.